src_ext/Makefile.packages

```diff
@@ -41,9 +41,13 @@ dose3.pkgbuild: findlib.pkgbuild ocamlbuild.pkgbuild cppo.pkgbuild extlib.pkgbui
 # mccs.pkgbuild depends on findlib because the files are explicitly installed using it
 mccs.pkgbuild: dune-local.pkgbuild cudf.pkgbuild findlib.pkgbuild
 
+stdlib-shims.pkgbuild: dune-local.pkgbuild
+
+dune-local.pkgbuild: findlib.pkgbuild
+
 opam-file-format-pkg-build:
 	dune build @install -p opam-file-format
-	dune install -p opam-file-format
+	dune install -p opam-file-format opam-file-format
```

Comment:
I'm curious, are these changes really required? Isn't that already implied by the `-p <pkg>` argument?

Reply:
It turns out they're not (I assumed so, too). You get an error with ocamlgraph complaining that `ocamlgraph_gtk.install` hasn't been built.

Reply:
I put that in for the other just for belt-and-braces!

Reply:
I see, interesting